src/js/actions/actionTypes.js

```diff
@@ -0,0 +1,6 @@
+export const SORT_MOVIES = 'SORT_MOVIES';
+export const SEARCHBY = 'SEARCHBY';
+export const SEARCH_PHRASE_CHANGE = 'SEARCH_PHRASE_CHANGE';
+export const LOAD_MOVIES = 'LOAD_MOVIES';
+export const LOAD_MOVIES_SUCCESS = 'LOAD_MOVIES_SUCCESS';
+export const LOAD_MOVIES_FAILED = 'LOAD_MOVIES_FAILED';
```

Comment:
```suggestion
export const LOAD_MOVIES_FAILED = 'LOAD_MOVIES_FAILED';
```
Maybe it would be better to keep them in small 1 object:

```
export const actionTypes = {
[...actions]
SEARCHBY: 'SEARCHBY'
}
```
then you'd import only 1 object and use keys of it in reducer like: 
actionTypes.SEARCHBY